recipes/pycomm3/LICENSE

```diff
@@ -0,0 +1,24 @@
+The MIT License (MIT)
```

Comment:
This file is part of the pypi package. No need to add it here.

Reply:
Great, wasn't aware of this. Removed it!